dotnet/src/IntegrationTests/Connectors/MistralAI/TextEmbedding/MistralAITextEmbeddingTests.cs

```diff
@@ -26,7 +26,7 @@ public MistralAITextEmbeddingTests()
             .Build();
     }
 
-    [Fact(Skip = "This test is for manual verification.")]
+    [Fact] // (Skip = "This test is for manual verification.")
```

Comment:
Is this intended to be enabled ?

Reply:
Yes, I'll keep them enabled in the feature branch otherwise I have to keep enabling and disabling with every PR. But there will be no way t merge to main with these enabled because they will fail so no worries of them ending up in main.